.github/workflows/build.yml

```diff
@@ -44,9 +44,64 @@ jobs:
       - name: Upload Coverage Report
         uses: codecov/codecov-action@v1
 
+  bdd-test:
+    needs: ['unit-test']
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        kubernetes: [v1.20.1]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14.7
+
+      - name: Setup Minikube-Kubernetes
+        uses: manusa/actions-setup-minikube@v2.3.0
+        with:
+          minikube version: v1.16.0
+          kubernetes version: ${{ matrix.kubernetes }}
+          github token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build images locally
+        run: make lvm-driver-image || exit 1;
+
+      - name: bootstrap
+        run: make bootstrap
+
+      - name: Running tests
+        run: ./ci/ci-test.sh
+
+  ansible:
+    runs-on: ubuntu-latest
+    needs: ['lint', 'unit-test', 'bdd-test']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push the ansible image
+        uses: docker/build-push-action@v2
+        with:
+          file: ./e2e-tests/Dockerfile
+          push: true
+          load: false
+          platforms: linux/amd64
+          tags: |
+            ${{ secrets.IMAGE_ORG}}/lvm-localpv-e2e:ci
```

Comment:
This can be made an env. So that we can default to openebs. Ref https://github.com/openebs/node-disk-manager/blob/master/.github/workflows/build.yml#L63